pwiz_tools/Skyline/TestData/PwizFileInfoTest.cs

```diff
@@ -102,6 +97,26 @@ public void TestTicChromatogram()
             VerifyTicChromatogram(testFilesDir.GetTestPath("160109_Mix1_calcurve_075" + ExtensionTestContext.ExtWatersRaw), 5108, 372494752);
         }
 
+        [TestMethod]
+        public void TestQcTraces()
+        {
+            const string testZipPath = @"TestData\PressureTracesTest.zip";
+
+            var testFilesDir = new TestFilesDir(TestContext, testZipPath);
+
+            using (var msDataFile = new MsDataFileImpl(testFilesDir.GetTestPath("PressureTrace1" + ExtensionTestContext.ExtAbWiff)))
+            {
+                var pressureTraces = msDataFile.GetQcTraces();
+
+                VerifyQcTrace(pressureTraces[0], "Column Pressure (channel 1)", 1148, 0, 9.558333, 1470, 210, "pressure", "psi");

```

Comment:
Better to refer to the string constants than have these values typed out in strings.